draft-packages/illustration/KaizenDraft/Illustration/Base.module.scss

```
@@ -1,8 +1,51 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+
 .wrapper {
   width: 100%;
   margin: 0;
 }
 
+.figure {
+  margin: 0;
+  position: relative;
+
+  // nested to get more specificity, beat out the generic button styles
+  .pausePlayButton {
+    opacity: 0%;
+    position: absolute;
+    right: 1rem;
+    bottom: 1rem;
+    /* stylelint-disable declaration-no-important */
+    background-color: $color-white !important;
+    border: 1px solid $color-gray-400;
+    transition: all $animation-duration-immediate;
+
+    @media (hover: none) and (pointer: coarse) {
+      opacity: 100%;
+    }
+
+    svg {
+      color: $color-purple-800;
+      opacity: 70%;
+    }
+
+    &:hover {
+      background-color: $color-gray-200 !important;
+
+      svg {
+        opacity: 100%;
+      }
+    }
+  }
+
+  &:hover {
```

Should just be able to do this I think

```suggestion
  &:hover,
  &:focus {
```